aleph-client/src/pallets/multisig.rs

```diff
@@ -71,3 +129,388 @@ impl<S: SignedConnectionApi> MultisigUserApi for S {
         self.send_tx(tx, status).await
     }
 }
+
+/// A group of accounts together with a threshold.
+#[derive(Clone, Eq, PartialEq, Debug)]
+pub struct MultisigParty {
+    signatories: Vec<AccountId>,
+    threshold: MultisigThreshold,
+}
+
+impl MultisigParty {
+    /// Create new party from `signatories` and `threshold`.
+    ///
+    /// `signatories` can contain duplicates and doesn't have to be sorted. However, there must be
+    /// at least 2 unique accounts. There is also a virtual upper bound - `MaxSignatories` constant.
+    /// It isn't checked here (since it requires client), however, using too big party will fail
+    /// when performing any chain interaction.
+    ///
+    /// `threshold` must be between 2 and number of unique accounts in `signatories`. For threshold
+    /// 1, use special method `MultisigUserApi::as_multi_threshold_1`.
+    pub fn new(signatories: &[AccountId], threshold: MultisigThreshold) -> AnyResult<Self> {
```

Comment:
in every other places `anyhow::Result` is used. Use it for consistency for now, we can add task to refactor it.

Reply:
sorry, I must have used it in one place by mistake and then copied it few times; changed to `anyhow::Result`